2.py

```diff
@@ -0,0 +1,7 @@
+mport json
```

Comment:
Хорошее решение!
Но в начале кода допустили небольшую орфографическую ошибку.
```suggestion
import json
```